src/tests/profiler/common/ProfilerTestRunner.cs

```diff
@@ -49,6 +51,18 @@ public class ProfilerTestRunner
                 envVars.Add("COMPlus_JITMinOpts", "0");
             }
 
+            if (profileeOptions.HasFlag(ProfileeOptions.ReverseDiagnosticsMode))
+            {
+                Console.WriteLine("Launching profilee in reverse diagnostics port mode.");
+                if (String.IsNullOrEmpty(reverseServerName))
+                {
+                    throw new ArgumentException();
+                }
+
+                envVars.Add("DOTNET_DiagnosticPorts", reverseServerName);
+                envVars.Add("DOTNET_DefaultDiagnosticPortSuspend", "1");
```

Comment:
This is what is causing you to have to send the resume command twice. When you configure a DiagnosticPort, the default mode is suspend. If you set `DOTNET_DefaultDiagnosticPortSuspend` to `1`, that makes the default listen port (the socket/named pipe that gets created by default) _also_ be suspended.

I think your intent was to just set `DOTNET_DiagnosticPorts` with a suspend configuration.

https://github.com/dotnet/diagnostics/blob/main/documentation/design-docs/ipc-protocol.md#diagnostic-ports

Reply:
Ok, thanks for the heads up. I thought I had to set both for some reason. Removing `DOTNET_DefaultDiagnosticPortSuspend` does mean I only have to issue ResumeRuntime on the reverse server port